### PR TITLE
Plotcontrol colors

### PR DESCRIPTION
**Pull request by stephenwhitmarsh**
Added color for plotcontrol to help distinguish different frequency bands

### CODE REVIEW

**Comment by robertoostenveld** [comment]
just for information: this was already used in the recent patches used for performances (Akusmata, Rennes), right?

File: module/plotcontrol/plotcontrol.py [hunk 1]
```diff
@@ -104,22 +104,26 @@
 inputcurve   = []
 
 # Create panels for each channel
-for iplot in range(len(input_name)):
+# for iplot in range(len(input_name)):
```
Comment (robertoostenveld) [inline module/plotcontrol/plotcontrol.py]: Please don't leave old code commented out like this.

**Comment by robertoostenveld** [comment]
this one seems to also contain the commits that you sent as separate PR #344.

If you merge this PR first, there is probably nothing left to merge in the other one. That is fine, so please go ahead. I did a quick review of the code changes for both PRs and it looks good to me.

File: module/plotcontrol/plotcontrol.py (hunk 1 above)
Comment (stephenwhitmarsh) [reply module/plotcontrol/plotcontrol.py]: Yes, sorry

**Comment by stephenwhitmarsh** [comment]
Yes, the colors were used in the previous couple of performances

**Pull request merged**